README.md

```diff
@@ -13,51 +13,50 @@ Implements a simple bus client to exchange JSON message on queues using AMQP ser
 
     const bus = require('node-amqp-bus');
 
-## Examples
+## API
 
-Create a client:
+### bus.createBusClient(url)
 
-    const client = yield bus.createBusClient(URL);
-    
-Close client
+Creates a client. Returns a `Promise` of the client.
 
-    client.close(); // return a promise
+    const client = yield bus.createBusClient(url);
 
-Publish an event to the bus:
+### client.close()
 
-    client.publish('your-exchange', 'the-key', message);
-
-Use the client to listen to the bus:
+Closes the client. Returns a `Promise`.
 
-    yield client.listen('your-exchange', 'the-queue', 'the-key', (message, callback) => {
+    client.close();
```

Comment:
in the previous function, you said it returned a Promise and put a yield in the example.
Why not put yield here too ?